ts/private/ts_lib.bzl

```diff
@@ -53,9 +53,10 @@ STD_ATTRS = {
         allow_files = True,
         mandatory = True,
     ),
-    "supports_workers": attr.bool(
+    # TODO: make it None-able bool once https://github.com/bazelbuild/bazel/issues/14434 is fixed.
+    "supports_workers": attr.int(
```

Comment:
is this already a  breaking change? What happens for users who are currently passing True/False here?

Reply:
yeah. this is a breaking change for those using ts_project_rule instead of the wrapper.  If that's not too bad, I could make this non-breaking by wrapping the `ts_project_rule` in a macro.